packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -810,11 +1134,70 @@ void registerPlugins(Registrar registrar) {
 }
 ''';
 
+const String _dartPluginRegisterWith = r'''
+      try {
+        {{dartClass}}.registerWith();
+      } catch (err) {
+        print(
+          '`{{pluginName}}` threw an error: $err. '
+          'The app may not function as expected until you remove this plugin from pubspec.yaml'
+        );
+      }
+''';
+// TODO(egarciad): Evaluate merging the web and desktop plugin registry templates.
```

Comment:
Is there an issue for this?

Reply:
https://github.com/flutter/flutter/issues/80406